### PR TITLE
SignalR loglevel nuance in javascript

### PR DESCRIPTION
**Pull request by janmarques**
Spent a couple hours stuck on this until I found the github issue. Might be appropriate to mention it here as well?

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/signalr/diagnostics.md](https://github.com/dotnet/AspNetCore.Docs/blob/55e0509513347b19cba0910d32a1f210534189e6/aspnetcore/signalr/diagnostics.md) | [Logging and diagnostics in ASP.NET Core SignalR](https://review.learn.microsoft.com/en-us/aspnet/core/signalr/diagnostics?branch=pr-en-us-30874) |

### CODE REVIEW

**Review by guardrex — APPROVED** [review]
Thanks @janmarques! 🚀 ... I'm simplifying it a bit to just the key point. I'll ping Brennan now to take look.

File: aspnetcore/signalr/diagnostics.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ When using the JavaScript client, you can configure logging options using the `c
 
 [!code-javascript[](diagnostics/logging-config-js.js?highlight=3)]
 
-To disable logging entirely, specify `signalR.LogLevel.None` in the `configureLogging` method.
+To disable logging entirely, specify `signalR.LogLevel.None` in the `configureLogging` method. Note that some logging related to WebSockets are emitted directly by the browser and can't be disabled via setting the log level or any other means.
```
Comment (BrennanConroy) [inline aspnetcore/signalr/diagnostics.md:75]: I don't think this is specific to WebSockets.

Comment (guardrex) [reply aspnetcore/signalr/diagnostics.md:75]: I'll fix it.

**Comment by guardrex** [comment]
@BrennanConroy ... How about the following?

> To disable framework logging, specify `signalR.LogLevel.None` in the `configureLogging` method. Note that some logging is emitted directly by the browser and can't be disabled via setting the log level.

**Review by BrennanConroy — APPROVED** [review]

**Pull request merged**